source/adminguide/management.rst

```diff
@@ -13,7 +13,7 @@
    specific language governing permissions and limitations
    under the License.
 
-
+.. _resource_tags:
```

Comment:
@JoaoJandre  you mean to use a link to this in [the new bit](https://github.com/apache/cloudstack-documentation/pull/365/files#diff-41b306eaea2583fed83b94ac7420d6ad6dedf0aa8476cb04eea1853655a07237R20)?

It does not seem to work in the generated docs.

Reply:
probably it needs a file scope ref in addition to the section?

Reply:
Yes, thats what I meant, when I built it locally it had the link... I guess I'm using a different Sphinx version :man_shrugging: 
I'll search another way to add this reference

Reply:
I've looked at other references like this one on the docs, I think that the problem is that I was using an underscore in the middle of the reference name, now it should work.

Reply:
@DaanHoogland I used the same logic as the `time-zones` reference, in the dev.rst file, there's the label:
```
.. _time-zones:
```
and in the usage.rst file, there's the reference:
```
Valid values for the time zone are specified in the :ref:`time-zones` section
```
Do you have any ideas on how to do this?

Reply:
apart from my remark in the refering section, is the issue maybe a cross file reference? I know these should be possible as well, but these might work slightly differently.

Reply:
The time-zones reference is also cross file, that's the weird part..